css/port.css

```diff
@@ -236,142 +336,325 @@ section .aim ul li {
   margin: 10px 0;
 }
 
-.btn-resume {
-  text-decoration: none;
+.pot1 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+.pot2 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+.pot3 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+/* ---Portfolio----- */
+@media (max-width: 470px) {
+  .simi {
+    display: block;
+    align-items: center;
+    padding: 15px;
+    margin-top: 100px;
+  }
+
+  .title {
+    height: auto;
+  }
+
+  .title-img {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .title-text {
+    margin-top: 45px;
+    padding-left: 20px;
+    gap: 12px;
+  }
+
+  .title-text1 {
+    margin-top: 50px;
+    padding-left: 24px;
+    gap: 24px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .title-img1 {
+    padding-right: 10px;
+    padding-left: 10px;
+  }
+
+  .t-text .port h4 {
+    color: #091e42;
+    font-weight: 700, bold;
+    font-size: 30px;
+    line-height: 21px;
+    margin-top: -30px;
+  }
+
+  .t-text .port ul {
+    max-width: 65%;
+    font-size: 15px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  .t-text ul li:first-child {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    color: #344563;
+  }
+
+  .q-text {
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    max-width: 85%;
+  }
+
+  .title1 {
+    height: auto;
+    width: auto;
+  }
+
+  .about {
+    max-width: 100%;
+    height: 924px;
+    position: absolute;
+    margin-top: 700px;
+  }
+
+  .about_header {
+    margin-left: 10px;
+    font-size: 40px;
+  }
 }
 
-.btn:disabled {
-  color: #5e6c84;
-  border: 1px solid #5e6c84;
+.about {
+  max-width: 100%;
+  height: 924px;
+  background-color: #fff;
+  top: 4535px;
+  border-top-right-radius: 90px;
 }
 
-.q-text {
-  font-size: 15px;
-  line-height: 24px;
+.win {
   color: #344563;
+  font-size: 16px;
+  line-height: 24px;
 }
 
-.arrow-right {
-  display: flex;
-  justify-content: end;
-  align-items: center;
-  margin-left: 18rem;
-  font-weight: normal;
+.left-block {
+  width: 600px;
+  height: 392px;
+  padding-left: 142px;
+  padding-top: 142px;
 }
 
-.ul .first-child {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-  color: #344563;
+.about_header {
+  font-weight: 700px;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  height: 114px;
 }
 
-.win {
-  color: #344563;
+.about_text {
+  font-weight: 400px;
   font-size: 16px;
   line-height: 24px;
+  color: #344563;
+  top: 126px;
 }
 
-.flex {
-  display: flex;
-  flex-direction: column;
+.aim {
+  padding-left: 614px;
+  margin-top: -270px;
 }
 
-.welcome {
-  width: 100%;
-  border-bottom-left-radius: 100px;
+.get-b {
+  padding-top: 35px;
 }
 
-.about {
-  width: 375px;
-  margin: 70px auto;
-  background-color: #fff;
-  border-top-right-radius: 100px;
-  padding: 100px 0;
-  height: 1263px;
-}
+/* -----aboutus------------ */
+@media (max-width: 470px) {
```

Comment:
- [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.